utils/logrotate/Makefile

```diff
@@ -9,22 +9,24 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=logrotate
 PKG_VERSION:=3.15.0
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:= https://github.com/logrotate/logrotate/releases/download/$(PKG_VERSION)
 PKG_HASH:=313612c4776a305393454c874ef590d8acf84c9ffa648717731dfe902284ff8f
-PKG_LICENSE:=GPL-2.0
+
+PKG_MAINTAINER:=Christian Beier <cb@shoutrlabs.com>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/logrotate
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+libpopt
+  DEPENDS:=+libpopt +USE_FS_ACL_ATTR:libacl
```

Comment:
I was thinking about doing it like rsync, with an explicit option, and `MENU:=1`.  It makes it easier to find than at the Kernel Config.  My reasoning is, apart from someone with kernel support not wanting to enable it for logrotate is that you may enable ACL in a fs by fs basis. Besides when a user needs this, the easiest/first thing to check would be the package, but I recon that could well be just baseless speculation on my part :smile:.  You may add a reference to the kernel options at the package help, so user can enable it in the kernel and actually use ACLs.  Anyway, you should add whatever symbol you're using to `PKG_CONFIG_DEPENDS`.

Reply:
I think this is what you meant.

Reply:
Exactly. :smiley: